src/main/java/mate/academy/Main.java

```diff
@@ -8,9 +8,8 @@
 public class Main {
 
     public static void main(String[] args) {
```

Comment:
I can see that you have changed the structure of the project
`mate.academy` were 2 packages, but now they are directories `mate/academy`, so let's change it back to how it was.

Reply:
> I can see that you have changed the structure of the project `mate.academy` were 2 packages, but now they are directories `mate/academy`, so let's change it back to how it was.
You propably should revise java specification